From 657db749b61055781063347467bf4bb456f0c2e4 Mon Sep 17 00:00:00 2001
From: vgdn1942 <vgdn1942@gmail.com>
Date: Fri, 12 Nov 2021 00:08:00 +0300
Subject: [PATCH 05/24] Remove Chrome

Change-Id: I4b523a795813c3e318e5b47b177611b20bb1985c
---
 gapps.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gapps.mk b/gapps.mk
index 15e12a9..744cf31 100644
--- a/gapps.mk
+++ b/gapps.mk
@@ -9,7 +9,6 @@ PRODUCT_PACKAGES := $(filter-out GoogleContacts, $(PRODUCT_PACKAGES))
 
 PRODUCT_PACKAGES += \
 	phh-gapps-overrides \
-	Chrome \
 	GoogleContactsSyncAdapter \
 	talkback \
 
-- 
2.25.1

